schemas/didDocument/didDocument.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$id": "/did-core.didDocument",
+    "type": "object",
+    "properties": {
+      "@context": {
+        "title": "@context",
+        "description": "https://w3c.github.io/did-core-registry/#context",
+        "anyOf": [
+          { "type": "string" },
+          { "type": "array" },
+          { "type": "object" }
+        ],
+        "required": false
+      },
+      "id": {
+        "title": "DID",
+        "description": "https://w3c.github.io/did-core-registry/#id",
+        "type": "string",
+        "required": true,
+        "example": ["did:key:z6MksQ35B5bwZDQq4QKuhQW2Sv6dcqwg4PqcSFf67pdgrtjB"]
+      },
+      "publicKey": {
+        "title": "Public Key",
+        "description": "https://w3c.github.io/did-core-registry/#publicKey",
+        "type": "array",
+        "required": true,
```

Comment:
publicKey isn't required, IIRC. If it is, it shouldn't be.

Reply:
You are correct, from the current spec: 

> If a DID document includes a publicKey property, the value of the property MUST be an array of public key objects. Each public key object MUST have the type, controller, and specific public key properties, and SHOULD have an id property. The public key objects MAY include additional properties.